docs/design/blocks/data-exporter.mdx

```diff
@@ -1,47 +1,274 @@
 ---
 title: "Data exporter"
 description:
-  "Once you’re done transforming your data, write code in these types of blocks to
-  store that data or to train models and store those models elsewhere."
+  "After completing data transformations, utilize the data exporter blocks to either load the processed 
+  data or store a machine learning model in an external data storage system."
 ---
 
-## Example
+<img
+  alt="exporter"
+  src="https://github.com/mage-ai/assets/blob/main/blocks/exporter/world-export.gif?raw=True"
```

Comment:
@colefreeman Can we change this gif to something without a company's logo? Just seems odd to have one with "eraplast electric" on it. And then ill approve the PR

Reply:
Fixed
![Uploading Screenshot 2024-07-02 at 3.37.59 PM.png…]()

Reply:
@johnson-mage deleted the files and made the change for the asset tag in the doc and it is appearing in mintlify dev env